modules/ROOT/pages/basic-authentication-ldap-concept.adoc

```diff
@@ -66,10 +100,12 @@ All fields are required. These literal string values support property placeholde
 |${ldap.search.filter}
 |===
 
-The search filter string given above is specific to Active Directory applications.
+The search filter string in the example is specific to Active Directory applications.
```

Comment:
What about using filter *value* or capitalizing String?

Reply:
Left it as is. I think it works.